test/active_job/queue_adapters/amazon_sqs_adapter_test.rb

```diff
@@ -188,6 +188,30 @@ module QueueAdapters
           end.to raise_error ArgumentError
         end
       end
+
+      if Gem::Version.new(Rails::VERSION::STRING) >= Gem::Version.new('7.1.0')
```

Comment:
One option we can consider is major version bumping aws-sdk-rails to hard depend on Rails 7.1, and dropping old dependencies. I would not be against this.

Reply:
This change has no effect on the old rails version, so it could be tagged as minor change. 
but if there is no other opportunity like this PR(supports Rails 7.1 feature), I think it would be a good idea to cut off old version's support and major bumping with this change.

Reply:
I'll talk with the team and get back to you.

Reply:
I think a minor version release is fine for this. Checking the dependencies, they're already loose.